Assets/Scripts/MazePlayerController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(Rigidbody))]
+
+public class MazePlayerController : MonoBehaviour
+{
+    private const string Horizontal = "Horizontal";
+    private const string Vertical = "Vertical";
+
+    private Rigidbody rb;
+
+    [Range(1.0f, 30f)]
+    public float speed;
```

Comment:
You didn't need to change this to public. You can do:

```c#
[Range(1f,30f)]
[SerializableField]
private float speed;
```

Also, a public variable should be capitalized. 
```